.circleci/deploy-to-pantheon.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -ex
+
+TERMINUS_DOES_MULTIDEV_EXIST()
+{
+    # Stash list of Pantheon multidev environments
+    PANTHEON_MULTIDEV_LIST="$(terminus multidev:list -n ${TERMINUS_SITE} --format=list --field=Name)"
+
+    while read -r multiDev; do
+        if [[ "${multiDev}" == "$1" ]]
+        then
+            return 0;
+        fi
+    done <<< "$PANTHEON_MULTIDEV_LIST"
+
+    return 1;
+}
+
+if [[ $CIRCLE_BRANCH == "master" ]]
+then
+    terminus build:env:push -n "$TERMINUS_SITE.$TERMINUS_ENV" --yes
+    terminus secrets:set -n "$TERMINUS_SITE.$TERMINUS_ENV" token "$GITHUB_TOKEN" --file='github-secrets.json' --clear --skip-if-empty
```

Comment:
I just had to remove the brackets around `if -n ${CIRCLE_PULL_REQUEST+x} && ! TERMINUS_DOES_MULTIDEV_EXIST ${TERMINUS_ENV}` and it was good to go.

See [this workflow](https://circleci.com/workflow-run/2823acdf-d27e-40cb-9a86-0bca8245a3c6) for an example of deployment to Pantheon and Behat being skipped